### PR TITLE
[feature](multi-catalog) add catalog comment and create time info

### PR DESCRIPTION
**Pull request by wsjz**
# Proposed changes

add catalog comment and create time info
```
create catalog hms_ctl
comment 'your comment' 
properties (
'type'='hms',
'hive.metastore.uris' = 'thrift://xx:1234' );
```
Create Time will generate when the catalog is created.

use show catalogs and show create catalog to get these info.

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/AlterCatalogPropertyStmt.java [hunk 1]
```diff
@@ -36,17 +36,23 @@
  */
 public class AlterCatalogPropertyStmt extends DdlStmt {
     private final String catalogName;
+    private final String comment;
     private final Map<String, String> newProperties;
 
     public AlterCatalogPropertyStmt(String catalogName, Map<String, String> newProperties) {
         this.catalogName = catalogName;
         this.newProperties = newProperties;
+        this.comment = newProperties.getOrDefault("comment", null);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/AlterCatalogPropertyStmt.java:45]: better use `""`, not `null`

File: fe/fe-core/src/main/java/org/apache/doris/analysis/CreateCatalogStmt.java [hunk 2]
```diff
@@ -32,24 +32,30 @@
...
     /**
      * Statement for create a new catalog.
      */
-    public CreateCatalogStmt(boolean ifNotExists, String catalogName, String resource, Map<String, String> properties) {
+    public CreateCatalogStmt(boolean ifNotExists, String catalogName, String resource, Map<String, String> properties,
+            String comment) {
         this.ifNotExists = ifNotExists;
         this.catalogName = catalogName;
         this.resource = resource == null ? "" : resource;
+        this.comment = comment == null ? "" : comment;
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/CreateCatalogStmt.java:58]: Strings.nullToEmpty()

**Comment by wsjz** [comment]
run buildall

**Comment by wsjz** [comment]
run buildall

**Comment by wsjz** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**